impose/flblend.py

```diff
@@ -95,19 +95,24 @@ def blend_rgb(self):
         stack_r = np.zeros((shape[0], shape[1], len(self)), dtype=float)
         stack_g = np.zeros((shape[0], shape[1], len(self)), dtype=float)
         stack_b = np.zeros((shape[0], shape[1], len(self)), dtype=float)
+        stack_v = np.zeros((shape[0], shape[1], len(self)), dtype=float)
 
         for ii, flim in enumerate(self.images):
             rgb = flim.get_rgb()
             stack_r[:, :, ii] = rgb[:, :, 0]
             stack_g[:, :, ii] = rgb[:, :, 1]
             stack_b[:, :, ii] = rgb[:, :, 2]
+            stack_v[:, :, ii] = np.invert(np.isnan(rgb[:, :, 0]))
+
+        stack_v = np.nansum(stack_v, axis=2)
```

Comment:
Please use `norm_v` instead of overriding `stack_v`  to make clear that you are using it for normalization. Also please add a comment that you are using it for normalization.

Reply:
Done.